sdk/core/Azure.Core/feature_lifecycle.md

```diff
@@ -0,0 +1,31 @@
+# Core Ship Sequence (draft)
+Legend:
+- diamond shapes == Phase
+- hexagonal shapes == Sub Step
+
+```mermaid
+flowchart TD
+    A{Feature Definition} --> B{{Feature Triage}}
+    B -->|Committed| SelS{{Select Shipping Sprint}}
+    SelS --> AtS{{Add Issue to Sprint Plan}}
+    AtS --> DI{Design and<Br/> Implementation}
+    DI --> DDes{{Document Design}}
```

Comment:
Do we want to show a step for developing the design?  We're also missing an implementation step, and I would expect some looping to happen between these two things.

Reply:
I'm hesitant to make this too granular. I think we should err on the side of providing the high level view of what is happening unless there is some significant and non-obvious decision branch that we want document about a sub-step.